packages/eslint-config/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@talend/eslint-config",
+  "version": "0.1.0",
+  "description": "Tools configurations",
+  "author": "Talend Frontend <frontend@talend.com>",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Talend/ui.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Talend/ui/issues"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "babel-eslint": "8.2.3",
+    "eslint": "4.19.1",
+    "eslint-config-airbnb": "16.1.0",
+    "eslint-plugin-import": "2.12.0",
+    "eslint-plugin-jsx-a11y": "6.0.3",
+    "eslint-plugin-react": "7.9.1"
```

Comment:
I'm puzzled that there's no mention of `jest` here Oo

Reply:
I don't understand why we should have. It's only about the linter and no usage of eslint jest plugin.

Reply:
The only reference to jest that we have is in the `env` section to indicate that jest is global